src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,37 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(health, name, hidden = false) {
```

Comment:
[CONSTRUCTOR PARAMETERS] - The constructor of `Herbivore` should not accept `health` as a parameter since it is always set to `100` by default in the `Animal` class. Remove the `health` parameter.